packages/language-server/src/completion/completions.ts

```diff
@@ -602,6 +602,14 @@ function getDefaultValues({
       break
   }
 
+  const isScalarList = fieldType.endsWith('[]')
+  if (isScalarList) {
+    return [
+      { label: '[]', kind: CompletionItemKind.Value },
+      { label: 'dbgenerated()', kind: CompletionItemKind.Function },
```

Comment:
We currently have a better completion for `dbgenerated()` here
https://github.com/prisma/language-tools/blob/main/packages/language-server/src/completion/completions.ts#L515-L522